src/parser/deathknight/blood/modules/features/Checklist.js

```diff
@@ -96,15 +96,16 @@ class Checklist extends CoreChecklist {
             check: () => this.runeTracker.suggestionThresholdsEfficiency,
           }),
           new Requirement({
-            name: <><SpellLink id={SPELLS.MARROWREND.id} /> efficiency</>,
+            name: <><SpellLink id={SPELLS.MARROWREND.id} /> Efficiency</>,
```

Comment:
imo no capital letter is correct here, since it isn't a title or name.

Reply:
I capitalized it because it looked weird when half of the `Efficiency`'s is lowercase and the others are uppercase.
And it appears like blood isn't the only one that's using it both ways.

![image](https://user-images.githubusercontent.com/29842841/48064904-86244480-e1c9-11e8-9f6e-36534328a902.png)
![image](https://user-images.githubusercontent.com/29842841/48064912-91777000-e1c9-11e8-9437-84bffcdaa3cf.png)